### PR TITLE
Fixed heartbeat to be 85% of session expiry

### PR DESCRIPTION
**Pull request by davidstl**
+ added or fixed unit tests

### CODE REVIEW

File: UnitTests/src/Main.as [hunk 1]
```diff
@@ -8,17 +8,18 @@ package
...
     public class Main extends Sprite 
     {
-        public var APP_ID:String = "";
-        public var SECRET:String = "";
-        public var APP_VERSION:String = "";
-        public var SERVER_URL:String = "";
-        public var PARENT_LEVEL_NAME:String = "";
-        public var CHILD_APP_ID:String = "";
-        public var PEER_NAME:String = "";
+        public var APP_ID:String = "20001";
```
Comment (stevemrjones) [inline UnitTests/src/Main.as]: you sohudln't put these in here :S

File: brainCloudClient/src/com/bitheads/braincloud/comms/BrainCloudComms.as [hunk 2]
```diff
@@ -626,11 +627,31 @@ package com.bitheads.braincloud.comms
...
+                    startHeartbeatTimer();
+                }
+            }
+        }
 		
 		private function startHeartbeatTimer():void
 		{
 			_heartbeatTimer.reset();
-			_heartbeatTimer.delay = _idleTimeout > 0 ? _idleTimeout * 1000 : 60000;
+            if (_desiredIdleTimeout == 0)
```
Comment (stevemrjones) [inline brainCloudClient/src/com/bitheads/braincloud/comms/BrainCloudComms.as]: Funky formatting

File: UnitTests/src/Main.as (hunk 1 above)
Comment (davidstl) [reply UnitTests/src/Main.as]: oops ya it shouldn't have made it. File sometimes doesn't get reverted when test fails. Will fix.

Good catch! thx.

File: brainCloudClient/src/com/bitheads/braincloud/comms/BrainCloudComms.as (hunk 2 above)
Comment (davidstl) [reply brainCloudClient/src/com/bitheads/braincloud/comms/BrainCloudComms.as]: ya I use spaces... will put back tabs there.

**Pull request merged**